src/node_lkl.cc

```diff
@@ -150,3 +148,39 @@ NAN_METHOD(syscall) {
 		}
 	}
 }
+
+NAN_METHOD(parseDirent64) {
```

Comment:
Same thing for listdir() in readdir.js.
It is not a library function.
It is a helper for readdir().
The only reason it is not contained in readdir() is because it has to be recursive.